specs/_features/maxeb_increase/capella.py

```diff
@@ -425,6 +427,19 @@ class SignedVoluntaryExit(Container):
     message: VoluntaryExit
     signature: BLSSignature
 
+class Consolidation(Container):
+    source_index: ValidatorIndex
```

Comment:
maybe make the PendingConsolidation a member of this class? to avoid duplication of src, targ pattern ? idk maybe better to keep separate to avoid nesting. up to you